src/ext/store.ts

```diff
@@ -17,6 +17,8 @@ import {
   selectedDatasetAndUserPlaceGroupsSelector,
   selectedVariable2NameSelector,
   selectedVariableNameSelector,
+  selectedDatasetTitleSelector,
+  selectedDatasetTitle2Selector,
```

Comment:
```suggestion
  selectedDataset2TitleSelector,
```

Reply:
Although not an issue to be addressed in this PR, please rename the selector `selectedDatasetTitle2Selector` --> `selectedDataset2TitleSelector`. It is the 2nd dataset, not the 2nd title.